### PR TITLE
Migrate to androidx, 2.0.0 release

### LINKED ISSUE #24: Can you please convert to androidx?

### PR DESCRIPTION
**Pull request by markushi**
Fixes #24

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -16,18 +16,18 @@
...
+        compileSdkVersion = 29
+        buildToolsVersion = '29.0.0'
 
         minSdkVersion = 21
-        targetSdkVersion = 28
+        targetSdkVersion = 29
 
         // App dependencies
-        androidGradlePluginVersion = '3.3.0-alpha05'
+        androidGradlePluginVersion = '3.4.2'
```
Comment (romtsn) [inline build.gradle]: 3.6.3 ? ;)

File: build.gradle [hunk 2]
```diff
@@ -16,18 +16,18 @@
 buildscript {
     ext {
         // SDK and tools
-        compileSdkVersion = 28
-        buildToolsVersion = '28.0.0'
+        compileSdkVersion = 29
+        buildToolsVersion = '29.0.0'
```
Comment (romtsn) [inline build.gradle]: '29.0.2' I believe

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```
Comment (romtsn) [inline gradle/wrapper/gradle-wrapper.properties]: could be 6.3

File: lib/src/main/java/com/github/okdroid/checkablechipview/CheckableChipView.kt [hunk 4]
```diff
@@ -368,7 +371,7 @@ class CheckableChipView @JvmOverloads constructor(
     }
 
     override fun onSaveInstanceState(): Parcelable {
-        return SavedState(super.onSaveInstanceState()).apply {
+        return SavedState(super.onSaveInstanceState()!!).apply {
```
Comment (markushi) [inline lib/src/main/java/com/github/okdroid/checkablechipview/CheckableChipView.kt:374]: Looking at the source `super.onSaveInstanceState()` can actually never return null, looks like a wrong @Nullable annotation

**Review by romtsn — APPROVED** [review]

File: lib/src/main/java/com/github/okdroid/checkablechipview/CheckableChipView.kt (hunk 4 above)
Comment (romtsn) [reply lib/src/main/java/com/github/okdroid/checkablechipview/CheckableChipView.kt:374]: [For child classes I guess](https://cs.android.com/android/_/android/platform/frameworks/base/+/517bafa59df45b838245252503b94fc929cf8844...a22a5d83c2ec9d4b0549e92c1d3af7550896e7ee)

**Pull request merged**